Bin/Snapshot.php

```diff
@@ -49,19 +49,34 @@
  */
 class Snapshot extends Console\Dispatcher\Kit
 {
+    /**
+     * User friendly labels for changelog themes.
+     *
+     * @const
+     */
+    public const CHANGELOG_THEMES = [
+        'chore' => 'Quality',
+        'fix'   => 'Bug fixes',
```

Comment:
I'm not sure about the vocabulary here. Do we need to rely on keepachangelog.com? Angular.JS uses the same vocabulary than yours. I'm fine with it. I'm just asking questions :-).

Reply:
A hard question 😄.

I think that Angular.js miss some important keywords : `Deprecated`, `Security` but add interesting ones...

Adding 2 more keywords will complexify the commit writing a bit, but we'll have a more robust changelog...

We'll finally have :

* `chore`,
* `fix`,
* `feat`,
* `depr` => Deprecated,
* `sec` => Security,
* `doc`,
* `test`,
* `undef`.

The keepachangelog.com keywords are too global for me, it's more important to reference the context of the update than the type.

Reply:
OK to add `depr` and `sec`. Do we need to update any documentation somewhere?

Reply:
You are right, I'll update the hoaproject/Literature#32 PR with these new keywords.